core/src/main/java/nl/nn/adapterframework/pipes/WsdlXmlValidator.java

```diff
@@ -323,13 +358,22 @@ public String toExtendedString() {
 		return "[" + getConfigurationClassLoader() + "][" + FilenameUtils.normalize(getWsdl()) + "][" + getSoapBody() + "][" + getOutputSoapBody() + "][" + getSoapBodyNamespace() + "]";
 	}
 
-	@IbisDoc({"1", "pairs of uri references which will be added to the wsdl", " "})
+	@IbisDoc({"the wsdl to read the xsd's from", " "})
+	public void setWsdl(String wsdl) {
+		this.wsdl = wsdl;
+	}
+
+	@IbisDoc({"Name of the child element of the SOAP body, or a comma separated list of names to choose from (only one is allowed) (wsdl generator will use the first element) (use empty value to allow an empty soap body, for example to allow element x and an empty soap body use: x,). In case the request contains SOAPAction header and the wsdl contains an element specific to that SOAPAction, it will use that element as soapBody.", "" })
```

Comment:
```suggestion
	@IbisDoc({"Name of the child element of the SOAP body, or a comma separated list of names to choose from (only one is allowed) (WSDL generator will use the first element) (use empty value to allow an empty SOAP body, for example to allow element x and an empty SOAP body use: x,). In case the request contains SOAPAction header and the WSDL contains an element specific to that SOAPAction, it will use that element as SOAP body.", "" })
```